tfx/components/transform/executor_test.py

```diff
@@ -72,24 +90,35 @@ def _get_output_data_dir(self, sub_dir=None):
 
   def _make_base_do_params(self, source_data_dir, output_data_dir):
     # Create input dict.
-    examples = standard_artifacts.Examples()
-    examples.uri = os.path.join(source_data_dir, 'csv_example_gen')
-    examples.split_names = artifact_utils.encode_split_names(['train', 'eval'])
+    e1 = standard_artifacts.Examples()
+    e1.uri = self._artifact1_uri
+    e1.split_names = artifact_utils.encode_split_names(['train', 'eval'])
+    e2 = copy.deepcopy(e1)
+    e2.uri = self._artifact2_uri
+
+    self._single_artifact = [e1]
```

Comment:
Instead of having single_artifact vs. multiple_artifacts, can we just have artifacts that the tests can choose to just use the first for example?